R/assocTestSingle.R

```diff
@@ -93,10 +106,25 @@ setMethod("assocTestSingle",
 setMethod("assocTestSingle",
           "GenotypeIterator",
           function(gdsobj, null.model, test=c("Score", "Score.SPA", "BinomiRare", "CMP"),
-                   recalc.pval.thresh=0.05, GxE=NULL,
+                   recalc.pval.thresh=0.05, approx.score.SE=FALSE, GxE=NULL,
                    male.diploid=TRUE, verbose=TRUE) {
               test <- match.arg(test)
 
+              # Convert old null model format if necessary.
+              null.model <- .updateNullModelFormat(null.model)
```

Comment:
@amstilp I'm not sure why this wasn't here from your null model updates (like it is for SeqVarIterators). Was it just a mistake? or should I drop it?

Reply:
I think I just missed it. It's also missing in the `assocTestAggregate` method for `GenotypeIterator` objects. Can you add it there too?

Reply:
added to assocTestAggregate